extensions/arc/package.nls.json

```diff
@@ -91,10 +91,21 @@
 	"arc.storage-class.data.label": "Storage Class (Data)",
 	"arc.sql.storage-class.data.description": "The storage class to be used for data (.mdf). If no value is specified, the default storage class will be used.",
 	"arc.postgres.storage-class.data.description": "The storage class to be used for data persistent volumes",
+	"arc.storage-class.datalogs.label": "Storage Class (Database logs)",
+	"arc.sql.storage-class.datalogs.description": "The storage class to be used for database logs (.ldf). If no value is specified, the default storage class will be used.",
 	"arc.storage-class.logs.label": "Storage Class (Logs)",
 	"arc.sql.storage-class.logs.description": "The storage class to be used for logs (/var/log). If no value is specified, the default storage class will be used.",
 	"arc.postgres.storage-class.logs.description": "The storage class to be used for logs persistent volumes",
 	"arc.storage-class.backups.label": "Storage Class (Backups)",
+	"arc.sql.storage-class.backups.description": "The storage class to be used for backup persistent volumes. If no value is specified, the default storage class will be used.",
+	"arc.sql.volume.size.data.label": "Volume Size in Gi (Data)",
+	"arc.sql.volume.size.data.description": "The size of the storage volume to be used for data in Gi.",
```

Comment:
For the description it would probably be best to spell out Gibibytes instead of using Gi in case people aren't familiar with the unit